docker-compose.yml

```diff
@@ -42,7 +42,15 @@ services:
       - 8088:80
     volumes:
       - ./bin/local-dev/wordpress/html:/var/www/html
-      - .:/var/www/html/wp-content/plugins/material-design
+      - ./plugin:/var/www/html/wp-content/plugins/material-design
+      - ./theme:/var/www/html/wp-content/themes/material-design-google
+      # Mount composer files and the vendor folder to both the plugin and theme folders.
+      - ./vendor:/var/www/html/wp-content/plugins/material-design/vendor
```

Comment:
Are we using the Composer autoloader in the final release?

Composer autoloader uses file paths relative to the directory of the `composer.json` file for resolving the dependencies so placing the `vendor` directory in a different folder relative to the project root will cause it to fail.

This is not an issue if we're not using the actual autoloader part of it.

Reply:
We don't use composer either for autoloading or any dependencies. The autoloading is handled by custom function in the [`class-plugin-base.php`](https://github.com/xwp/material-design-wp-plugin/blob/e0f797fce7de8aea124068293c17fea74401fe77/plugin/php/class-plugin-base.php#L129) file.